integrations/gcp/gcp_core/overrides.py

```diff
@@ -17,7 +17,24 @@ class GCPCloudResourceConfig(ResourceConfig):
     selector: GCPCloudResourceSelector
 
 
+class GCPResourceSelector(Selector):
+    preserve_api_response_case_style: bool | None = Field(
+        default=None,
```

Comment:
```suggestion
    preserve_api_response_case_style: bool = Field(
        default=False,
```

Reply:
lets let it be truly optional. You dont even have to pass it.

Reply:
That's if we didn't have to send anything at all to https://github.com/port-labs/ocean/blob/0c84972f32adbdb34fef0d74abbf6691c24542cc/integrations/gcp/gcp_core/utils.py#L93

Its the reason you had to set default within the code itself `False` 👇🏾

https://github.com/port-labs/ocean/blob/0c84972f32adbdb34fef0d74abbf6691c24542cc/integrations/gcp/gcp_core/utils.py#L84

Reply:
But its a minimal change, nothing impacting, still works either ways 💪🏽

Reply:
Fair point